emails/file-expiry.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+    Hr,
+    Link,
+    Section,
+    Text
+} from "@react-email/components";
+import { z } from "zod";
+import { WATcloudEmail } from "./_common/watcloud-email";
+
```

Comment:
As of the last PR, the new changes were to:

change the WatcloudEmail type back to WatcloudEmail, where I had previously changed it just to Email

Change the FileExpiryEmail component name to just Email

Changed how it lists the expired filepaths to just the strings itself, rather than separating them by machine name